package.json

```diff
@@ -41,8 +41,18 @@
   ],
   "main": "./out/src/extension",
   "contributes": {
-    "commands": [
-      {
+    "configuration": {
+      "type": "object",
+      "title": "quicktype",
+      "properties": {
+        "quicktype.inferMaps": {
+          "type": "boolean",
+          "default": false,
```

Comment:
Default should be `true`

Reply:
Default being `true` will expose the user to the error reported in https://github.com/quicktype/quicktype/issues/924 and the extension will not work.

If you guys plan to fix that issue very soon, It's okay to be `true`. Otherwise, the VSCode Extension in many cases will be useless.

Cheers

Reply:
I believe our default behavior on CLI and web is to infer maps, so I don't want it to be different here. Our false positive rate is lower than map occurrence in practice.

Reply:
Also, false positives for maps still produce working if awkward code; failing to identify maps produces runtime failurse for new map keys.

Reply:
Ok, feel free to edit that line. Hope you guys fix that issue and it propagates to all dependent tools.

Best